README_SYCL.md

```diff
@@ -97,7 +97,8 @@ These can be used interchangeably (some restrictions apply - see below) with the
   See [Intel's FAQ](https://github.com/intel/compute-runtime/blob/master/opencl/doc/FAQ.md#feature-double-precision-emulation-fp64) for more information.
 * The FPGA back-end does not support atomics. alpaka will not check this.
 * Device global variables (corresponding to `__device__` and `__constant__` variables in CUDA) are not supported in the SYCL back-end yet.
-* Shared memory does not yet work properly on CPU back-ends. The tests that use it (`atomicTest`, `blockSharedTest` and `blockSharedSharingTest`) fail with a `PI_ERROR_OUT_OF_RESOURCES`.
+* Shared memory works but on the GPU it's very slow.
```

Comment:
```suggestion
* Shared memory works but on the GPU it is very slow.
```

Is the reason for this alpaka's implementation or is this a general Intel thing?

Reply:
the alpaka implementation of the shared memory is slower than the native one

Reply:
yes, we are allocating a lot of shared memory for each kernel even if we do not use it and this causes a slow down, see #2068